webapp/controller/PageOverview.controller.js

```diff
@@ -0,0 +1,199 @@
+sap.ui.define([
+	"sap/ui/core/mvc/Controller",
+	'sap/ui/model/json/JSONModel',
+	"sap/ui/model/Filter",
+	"sap/ui/model/FilterOperator",
+	'sap/m/Token',
+], function (Controller, JSONModel, Filter, FilterOperator, Token) {
+	"use strict";
+
+	return Controller.extend("products.app.controller.PageOverview", {
+		
+		onInit: function() {
+			const oModel = new JSONModel();
+			oModel.loadData("../model/products.json");
+		
+			this.getView().setModel(oModel);
+		},
+
+		onAfterRendering: function() {
+			const oMultiInput = this.byId("multiInput");
+			oMultiInput.attachTokenUpdate((event) => {
```

Comment:
Why not define tokenUpdate event in xml view and attach handler from controller to it?

Reply:
I didn't use it, because in documentation updateToken like deprecated, and I thought that it is could be a mistake